packages/fluent-ui/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import MuiForm from "./MuiForm/MuiForm";
+import { initializeIcons } from "@fluentui/react";
+
+export { default as Fields } from "./Fields";
+export { default as FieldTemplate } from "./FieldTemplate";
+export { default as MuiForm } from "./MuiForm";
+export { default as ObjectFieldTemplate } from "./ObjectFieldTemplate";
+export { default as Theme } from "./Theme";
+export { default as Widgets } from "./Widgets";
+
+initializeIcons();
```

Comment:
Question: Does the order of this call in relation to the exports matter?

Reply:
Changed the order so all the import export statements go together.